problems/objects.js

```diff
@@ -61,7 +71,10 @@ const appleSet = () => {};
  *
  */
 
-const adamAndEveApples = () => {};
+const adamAndEveApples = (appleCountByName) => {
+  let sum = appleCountByName.Eve + appleCountByName.Adam;
```

Comment:
You can just return the code on line 75 without setting it to the variable sum and then returning sum. The code will be cleaner without the sum variable.